pacsman/dcmtk_client.py

```diff
@@ -80,7 +82,7 @@ def __init__(self, client_ae, remote_ae, pacs_url, pacs_port, dicom_dir, timeout
                          *self.logger_args,
                          '--output-directory', self.dicom_tmp_dir,
                          '--filename-extension', '.dcm',
-                         '--config-file', storescp_config_path, 'AllDICOM',
```

Comment:
Just curious about where the "AllDICOM" string originated from? I cannot find it in the dcmtk documentation

Reply:
The `AllDICOM` profile is included as a part of the default `storescp.cfg` file packaged with dcmtk. On my local machine I found the default file installed at `/usr/local/etc/dcmtk/storescp.cfg`